### PR TITLE
New integrations

### PR DESCRIPTION
**Pull request by dafimor**
@refaelos pls review, thanks.

### CODE REVIEW

File: src/documents/unity/grow/GrowIntegration_Facebook.html.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+collection: 'unity_grow'
+module: 'grow'
+platform: 'unity'
+---
+
+# Integrating Grow with Facebook
+
+## Overview
+
+ GROW is SOOMLA's flagship - a community-driven data network. Mobile game studios can take advantage of the different GROW products in order to get valuable insights about their games' performance and increase retention and monetization. [Read more...](/university/articles/Grow_About)
```
Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor GROW is soomla's flagship? It's not good english.

File: src/documents/unity/grow/GrowIntegration_Facebook.html.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### New Game & Configurations
+
+Go to the [GROW dashboard](http://dashboard.soom.la) and sign up \ login. Upon logging in, you will be directed to the main page of the dashboard. You will need to create a new game in order to start your journey with GROW.
+
+1. In the games screen click on the "+" button to add a new game. If it's your first time in the dashboard, just click on the "+" button underneath the "Create your first game" label in the middle of the screen.
+
+	  ![alt text](/img/tutorial_img/unity_grow/addNewApp.png "Add new app")
+
+	Once you created your game, you'll be redirected to a quick start process to download any of the GROW bundles (You can also click "Downloads" on the top right corner of the screen). Click on **Facebook**. You'll see an instructions screen, you can continue with that or stay here for the extended version.
```
Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor there's no Downloads link anymore there. Please go over the dashboard and fix in other documents if needed.

Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor also there are no bundles. We now have integrations.

**Comment by dafimor** [comment]
@refaelos updated according to your comments and fixed a few more places (where it said Bundles and the Download link that doesn't exist anymore). Thanks

File: src/documents/unity/grow/GrowIntegration_Facebook.html.md (hunk 2 above)
Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor also there's no quick start process

File: src/documents/unity/grow/GrowIntegration_Facebook.html.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+	<div class="info-box">The "game" and "environment" keys allow GROW to distinguish between multiple environments of your games. The dashboard pre-generates two fixed environments for your game: **Production** and **Sandbox**. When you decide to publish your game, **make sure to switch the environment key to <u>Production</u>**.  You can always generate more environments:  For example - you can choose to have a playground environment for your game's beta testers which will be isolated from your production environment and will thus prevent analytics data from being mixed between the two.  Another best practice might be to have a separate environment for each version of your game.</div>
+
+	<img src="/img/tutorial_img/unity_grow/dashboardKeys.png" alt="Game key and Env key" style="border:0;">
+
+	b. Click on the **Add Prefab** button and the prefab will be added to the **current scene**.
+	<div class="info-box"> Make sure that you are in your Main Scene when adding the Prefab </div>
```
Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor there's no closing paragraph.

File: src/documents/unity/grow/GrowIntegration_Facebook.html.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+collection: 'unity_grow'
+module: 'grow'
+platform: 'unity'
+---
+
+# Integrating Grow with Facebook
+
+## Overview
+
+ GROW is a community-driven data network. Mobile game studios can take advantage of the different GROW products in order to get valuable insights about their games' performance and increase retention and monetization. [Read more...](/university/articles/Grow_About)
```
Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor GROW is SOOMLA's flagship, community driven, data network....

File: src/documents/unity/grow/GrowIntegration_Facebook.html.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+### New Game & Configurations
+
+Go to the [GROW dashboard](http://dashboard.soom.la) and sign up \ login. Upon logging in, you will be directed to the main page of the dashboard. You will need to create a new game in order to start your journey with GROW.
+
+1. In the games screen click on the "+" button to add a new game. If it's your first time in the dashboard, just click on the "+" button underneath the "Create your first game" label in the middle of the screen.
+
+	  ![alt text](/img/tutorial_img/unity_grow/addNewApp.png "Add new app")
+
+	Once you created your game, you'll be redirected to an Integrations page where you can start with any of the GROW integrations. Click on **Facebook**. You'll see an instructions screen, you can continue with that or stay here for the extended version.
```
Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor ... you'll be redirected to **the** integrations page...

Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor ... you'll see **the** instructions...

File: src/documents/unity/grow/GrowIntegration_Facebook.html.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Go to the [GROW dashboard](http://dashboard.soom.la) and sign up \ login. Upon logging in, you will be directed to the main page of the dashboard. You will need to create a new game in order to start your journey with GROW.
+
+1. In the games screen click on the "+" button to add a new game. If it's your first time in the dashboard, just click on the "+" button underneath the "Create your first game" label in the middle of the screen.
+
+	  ![alt text](/img/tutorial_img/unity_grow/addNewApp.png "Add new app")
+
+	Once you created your game, you'll be redirected to an Integrations page where you can start with any of the GROW integrations. Click on **Facebook**. You'll see an instructions screen, you can continue with that or stay here for the extended version.  
+
+2. Double-click on the downloaded Unity package, it'll import all the necessary files into your Unity project.
```
Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_Facebook.html.md]: @dafimor Download and import the unitypackage...

File: src/documents/unity/grow/GrowIntegration_UnityIAP.html.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
```
Comment (refaelos) [inline src/documents/unity/grow/GrowIntegration_UnityIAP.html.md]: All issues from the facebook screen apply here as well.

File: src/documents/unity/grow/GrowSoomla_GettingStarted.html.md [hunk 8]
```diff
@@ -34,7 +34,7 @@ Go to the [GROW dashboard](http://dashboard.soom.la) and sign up \ login. Upon l
 
 	  ![alt text](/img/tutorial_img/unity_grow/addNewApp.png "Add new app")
 
-	Once you created your game, you'll be redirected to a quick start process to download any of the GROW bundles (You can also click "Downloads" on the top right corner of the screen). Click on **GrowSoomla**. You'll see an instructions screen, you can continue with that or stay here for the extended version.  
+	Once you created your game, you'll be redirected to an Integrations page where you can start with any of the GROW integrations. Click on **GrowSoomla**. You'll see an instructions screen, you can continue with that or stay here for the extended version.
```
Comment (refaelos) [inline src/documents/unity/grow/GrowSoomla_GettingStarted.html.md]: @dafimor ... the integrations page...

**Comment by refaelos** [comment]
@dafimor 2 more things:  
1.  Change the names of "Grow with Facebook" and "Grow with UnityIAP" to "Facebook" and "UnityIAP".
2. The "Grow Keys" image is incorrect. It's from the older dashboard.

**Comment by dafimor** [comment]
@refaelos fixed these pages.

**Comment by refaelos** [comment]
@dafimor yalla merge

**Comment by dafimor** [comment]
@refaelos I updated what we talked about, can I merge and continue?

**Pull request merged**